corehq/apps/app_manager/views/releases.py

```diff
@@ -470,7 +471,19 @@ def post(self, request, domain, app_id, *args, **kwargs):
                 id = profile.get('id')
                 if not id:
                     id = uuid.uuid4().hex
-                build_profiles[id] = BuildProfile(langs=profile['langs'], name=profile['name'])
+                def practice_user_id():
+                    if not app.enable_practice_users:
+                        return ''
+                    try:
+                        practice_user_id = profile.get('practice_user_id')
+                        if practice_user_id:
+                            get_and_assert_practice_user_in_domain(practice_user_id, domain)
+                        return practice_user_id
+                    except PracticeUserException:
+                        return HttpResponse(status=400)
```

Comment:
This should never really happen, it's just here to make the view safe from modified POSTs